src/arv.h

```diff
@@ -54,6 +54,7 @@
 #include <arvfakestream.h>
 
 #include <arvfeatures.h>
+#include <arvparams.h>
```

Comment:
Don't include the parameter header in arv.h.

Reply:
Done